src/main/kotlin/sipura/alg/Connectivity.kt

```diff
@@ -187,6 +187,9 @@ object Connectivity {
      * A vertex v is a cut-vertex if the graph has more connected components after v is removed from it.
      * An edge (v1,v2) is a bridge-edge if the graph has more connected components after (v1,v2) is removed from it.
      *
+     * Uses an algorithm that calculates a dfs-tree and a chain decomposition based on that tree. The cut-vertices
+     * and bridge-edges can then easily be determined from the chain decomposition.
+     *
```

Comment:
I haven't heard of "chain decompositions" so far.
The first results I found were https://en.wikipedia.org/wiki/Tree_decomposition and http://ktiml.mff.cuni.cz/~gregor/hypercube/lecture17.pdf, which both don't seem to fit.

Maybe add a reference to what you mean?